drivers/bluetooth/hci/h4.c

```diff
@@ -525,11 +536,24 @@ static int h4_open(void)
 	return 0;
 }
 
+static int h4_setup(void)
+{
+#if defined(CONFIG_BT_HCI_H4_SETUP)
+	/* h4_vnd_setup function must be implemented in vendor-specific HCI
+	 * extansion module. The option CONFIG_BT_HCI_H4_SETUP should be enabled.
+	 */
+	return h4_vnd_setup(h4_dev);
+#else
+	return 0;
+#endif
+}
+
 static const struct bt_hci_driver drv = {
 	.name		= "H:4",
 	.bus		= BT_HCI_DRIVER_BUS_UART,
 	.open		= h4_open,
 	.send		= h4_send,
+	.setup		= h4_setup
```

Comment:
#if defined(CONFIG_BT_HCI_H4_SETUP)

Reply:
after update https://github.com/zephyrproject-rtos/zephyr/pull/41453#discussion_r776332286 , it does not make sense to have #if... here.

now we always have h4_setup:
```
static int h4_setup(void)
{
#if defined(CONFIG_BT_HCI_H4_SETUP)
	/* h4_vnd_setup function must be implemented in vendor-specific HCI
	 * extension module. The option CONFIG_BT_HCI_H4_SETUP should be enabled.
	 */
	return h4_vnd_setup(h4_dev);
#else
	return 0;
#endif
}
```

Reply:
added #if defined(CONFIG_BT_HCI_H4_SETUP)..